config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -177,6 +177,14 @@ config CHIP_NFC_COMMISSIONING
     bool
     default y
 
+config CHIP_FACTORY_DATA
+    bool
+    default y
+
+config CHIP_FACTORY_DATA_BUILD
+    bool
```

Comment:
Do we want to build it by default? (I thought the decision was to make it in a separate PR :))